### PR TITLE
Fix watched null values.

### PR DESCRIPTION
**Pull request by p0psicles**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

@sharkykh still not sure if this PR is needed at all. But throwing it out here, just in case.

### CODE REVIEW

File: medusa/databases/main_db.py [hunk 1]
```diff
@@ -221,6 +222,10 @@ def fix_subtitles_codes(self):
     def fix_show_nfo_lang(self):
         self.connection.action("UPDATE tv_shows SET lang = '' WHERE lang = 0 OR lang = '0';")
 
+    def fix_tv_episodes_watched_field(self):
+        """A mistake was made when we added this field. Migration didn't went very well."""
```
Comment (OmgImAlexis) [inline medusa/databases/main_db.py]: Might want to swap "went" with "go".

**Comment by sharkykh** [comment]
Where have you seen `null` values?

**Comment by sharkykh** [comment]
@p0psicles 
IMO we really have to put a hold on migrations until the migration issues are solved...
I still not sure about this fix too. I mean it should fix it...

**Comment by p0psicles** [comment]
For new users yes. Users that are allready on db 4.12 not.

**Comment by sharkykh** [comment]
@p0psicles 
Why? I think sanity checks run every time Medusa starts up?

**Comment by sharkykh** [comment]
I don't know why I didn't think of this before.
`Episode.watched` should be boolean anyway, so we should be using `bool` on the value.

**Comment by p0psicles** [comment]
I also cant merge it

**Comment by p0psicles** [comment]
Yeah the bool does the trick

**Pull request merged**